olp-cpp-sdk-dataservice-read/src/generated/api/QueryApi.cpp

```diff
@@ -134,6 +134,33 @@ QueryApi::QuadTreeIndexResponse QueryApi::QuadTreeIndex(
   return olp::parser::parse<model::Index>(response.response);
 }
 
+olp::client::HttpResponse QueryApi::CreateQuadTreeIndex(
+    const client::OlpClient& client, const std::string& layer_id,
+    int64_t version, const std::string& quad_key, int32_t depth,
+    boost::optional<std::vector<std::string>> additional_fields,
+    boost::optional<std::string> billing_tag,
+    client::CancellationContext context) {
+  std::multimap<std::string, std::string> header_params;
+  header_params.insert(std::make_pair("Accept", "application/json"));
```

Comment:
```suggestion
  header_params.emplace("Accept", "application/json");
```
Same for below inserts also.